types/react-native-sensor-manager/index.d.ts

```diff
@@ -0,0 +1,44 @@
+// Type definitions for react-native-sensor-manager 0.1
+// Project: https://github.com/kprimice/react-native-sensor-manager
+// Definitions by: Sahin Vardar <https://github.com/SahinVardar>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="react-native" />
+
+export * from 'react-native'
+
+interface SensorManager {
+    startAccelerometer(delay: number): void;
+    stopAccelerometer(): void;
+    startGyroscope(delay: number): void;
+    stopGyroscope(): void;
+    startMagnetometer(delay: number): void;
+    stopMagnetometer(): void;
+    startStepCounter(delay: number): void;
+    stopStepCounter(): void;
+    startThermometer(delay: number): void;
+    stopThermometer(): void;
+    startMotionValue(delay: number): void;
+    stopMotionValue(): void;
+    startOrientation(delay: number): void;
+    stopOrientation(): void;
+    startProximity(delay: number): void;
+    stopProximity(): void;
+    startLightSensor(delay: number): void;
+    stopLightSensor(): void;
+}
+
+declare module 'react-native' {
+    export interface DeviceEventEmitterStatic {
```

Comment:
The original definition in react-native uses a template parameter which will be used for the data type.  Which looks like this.

`
 addListener<T>( type: string, listener: ( data: T ) => void, context?: any ): EmitterSubscription;
`

So it does not conflict with the original definition, it only augments the definition with non templated specialisation based on the even type.